starfish/imagestack/imagestack.py

```diff
@@ -1121,3 +1159,37 @@ def un_squeeze(self, stack):
         new_shape = (self.num_rounds, self.num_chs, self.num_zlayers) + self.tile_shape
         res = stack.reshape(new_shape)
         return res
+
+
+class _StripArguments:
+    """
+    Class to strip out arguments to an argument call.  This is spiritually the opposite of
+    functools.partial.  When this proxy is called, it removes some arguments and passes it to the
+    original callable.
+
+    This is used in scenarios where factory methods and lambdas cannot be used, such as dispatching
```

Comment:
I think I'm looking for examples of usage so that I can figure out why this is needed.

Reply:
simply put: one cannot pass a lambda or a method returned from another method because lexical scopes cannot be pickled.